src/plugin/elementSpec.ts

```diff
@@ -35,7 +35,7 @@ export type ValidationError = {
 export type FieldValidationErrors = Record<string, ValidationError[]>;
 
 export type Validator<FDesc extends FieldDescriptions<string>> = (
-  fields: FieldNameToValueMap<FDesc>
+  fields: Partial<FieldNameToValueMap<FDesc>>
```

Comment:
Torn – would it be worth adding a comment to indicate why this is a partial?

Reply:
I think that's fair, it's not clear at a glance why that might be.